packages/core/src/utils/configMergerFactory.ts

```diff
@@ -0,0 +1,25 @@
+import * as deepmerge from 'deepmerge';
```

Comment:
Is there a benefit of `deepmerge` versus `lodash.merge`? Asking simply because we generally stick to `lodash` ecosystem in the rest of the Stoplight ecosystem.

Reply:
@marbemac oh, not really, I didn't want to clog the dependencies with lodash (it's felt an overkill to include a whole library for 'just' deep merge.) I'm happy to swap to lodash if that's been the practice.